boot/boot_serial/src/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
 zephyr_sources(boot_serial.c)
 
 zephyr_include_directories(${BOOT_DIR}/bootutil/include)
+
+zephyr_include_directories($ENV{ZEPHYR_BASE}/ext/lib/mgmt/mcumgr/cborattr/include)
```

Comment:
@carlescufi I'm aware of that cborattr should be extracted to a zephyr module. I can do this after this patch  (which I prefer).